Projects/UOContent/Commands/Handlers.cs

```diff
@@ -21,7 +21,7 @@ public static class CommandHandlers
     {
         public static void Initialize()
         {
-            CommandSystem.Prefix = "[";
+            CommandSystem.Prefix = ServerConfiguration.GetSetting("command.prefix", "[");
```

Comment:
Let's use `GetOrUpdateSetting` so we can set a default in the config file so people can modify it there.

Reply:
Good idea, changing it! Thanks for feedback!